manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
I would now solve it like this in the file **server.pp**:
```
class dns::server {
  class { 'dns::server::install': } ->
  class { 'dns::server::config': } ->
  class { 'dns::server::default': } ~>
  class { 'dns::server::service': }
}
```
Do you agree with this changement?